### DISCUSSION TITLE
Vulnerability in Adguard Home

### DISCUSSION THREAD

**Discussion by Virus900** [AdguardTeam/AdGuardHome#3311] [Q&A]
Ubuntu 21.04 All update Adguard Home 0.0.106.3

Reproduction steps:

1. Install Adguard Home
2. Set up Adguard Home
3. Launched DNS Server 

Actual result:
Attack DDoS to my DNS Server type DNS Amplification.

Expected Result: 
Attack DDoS to DNS Server do not pass.

**Comment by ameshkov** [comment]
Here's what you can do:

1. Configure rate limit (it's rather low by default).
2. Use "Access settings" to control which clients are allowed to use your DNS server.

**Reply by Virus900** [reply]
I need it to be available to everyone. through unbound not Attacks DNS Amplification
